demo-projects/blog/app/pages/index.js

```diff
@@ -41,7 +42,7 @@ const Post = ({ post }) => {
   );
 };
 
-export default function() {
+export default withApollo()(() => {
```

Comment:
[2/2] this is the syntax soup I'm talking about, `withApollo()(()` could be simplified to `withApollo(()` because there's nothing to the first function invocation